bar-trender-front-end/src/components/Navbars/ExamplesNavbar.js

```diff
@@ -0,0 +1,168 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import barTrender from '../../assets/img/barTrender60.png';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/BarTrender/BarTrenderFront/pullRequest?prid=7164324)
```suggestion
import barTrender from "../../assets/img/barTrender60.png";
```